x-pack/plugins/observability_solution/apm/server/routes/dependencies/get_top_dependency_spans.ts

```diff
@@ -98,7 +97,6 @@ export async function getTopDependencySpans({
               ...kqlQuery(kuery),
               ...termQuery(SPAN_DESTINATION_SERVICE_RESOURCE, dependencyName),
               ...termQuery(SPAN_NAME, spanName),
-              { exists: { field: TRANSACTION_ID } },
```

Comment:
should the filter here check if `trace.id` exists to limit the scope of the query? Does that make sense?

Reply:
Not sure if that does a different impact, but the filter can be added

Reply:
Me neither, we can leave it out.